charts/latest/blob-csi-driver/templates/csi-blob-node.yaml

```diff
@@ -88,10 +88,14 @@ spec:
               value: "{{ .Values.node.blobfuseProxy.installBlobfuse }}"
             - name: BLOBFUSE_VERSION
               value: "{{ .Values.node.blobfuseProxy.blobfuseVersion }}"
+            - name: ALLOW_BLOBFUSE_DOWNGRADES
```

Comment:
blobfuse install is already deprecated, pls remove this `ALLOW_BLOBFUSE_DOWNGRADES` option